lib/inline_svg.rb

```diff
@@ -36,7 +36,7 @@ def add_custom_transformation(options)
       if incompatible_transformation?(options.fetch(:transform))
         raise InlineSvg::Configuration::Invalid.new("#{options.fetch(:transform)} should implement the .create_with_value and #transform methods")
       end
-      @custom_transformations.merge!(Hash[ *[options.fetch(:attribute, :no_attribute), options.fetch(:transform, no_transform)] ])
+      @custom_transformations.merge!(Hash[ *[options.fetch(:attribute, :no_attribute), options] ])
```

Comment:
hmm, I think we don't. I like the hash `attribute => options` struct, it is easy to (iterate / understand) and has a good performance in https://github.com/jamesmartin/inline_svg/pull/37/files#diff-cd6002cbfc3ddecb10a15ac6d5b394f5R28